charts/mercure/values.yaml

```diff
@@ -28,6 +28,22 @@ subscriberJwtAlg: HS256
 # -- The license key for [the High Availability version](https://mercure.rocks/docs/hub/cluster) (not necessary is you use the FOSS version).
 license: ""
 
+metrics:
+  # -- Enable the metrics. You must also add the [metrics](https://caddyserver.com/docs/caddyfile/directives/metrics) in the `globalOptions` for Caddy server.
+  # servers {
```

Comment:
Can't we add this option automatically using the by appending the needed config to the `GLOBAL_OPTIONS` environment variable in `deployment.yaml`?

Reply:
Indeed, let's do that like this for now. We could do something smarter in the future (like detection if `servers` is not part of the existing string, but we'll see in follow-up PRs!